api/dp-openapi.yaml

```diff
@@ -690,6 +690,162 @@ paths:
+        in: query
+        schema:
+          type: string
+      - name: group
+        description: Release queue group name substring to match. If not set all groups are
+          returned (as well as all matching queues in those groups).
+          <br><br>
+          To match multiple possible group names (results ORed together), append with []
+          and pass one per desired name (e.g. group[]=PM&group[]=Math).
+        in: query
+        schema:
+          type: string
+
+      responses:
+        200:
+          description: Requested release queue list
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/release_queue_detail'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '429':
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+
+  /queues/{queueid}:
+    get:
+      tags:
+      - queues
+      description: Get release queue info, release stats, and projects
+      parameters:
+      - name: queueid
+        in: path
+        description: Queue ID
+        required: true
+        schema:
+          type: integer
+      - name: unheld_only
+        in: query
+        schema:
+          type: boolean
+      - name: show_queue
+        in: query
+        schema:
+          type: boolean
+      - name: show_stats
+        in: query
+        schema:
+          type: boolean
+      - name: show_projects
+        in: query
+        schema:
+          type: boolean
+      - name: proj_field
+        description: Project field to return, if not set all fields are returned.
+          <br><br>
+          To request multiple fields, append with [] and pass one per desired
+          field (e.g. proj_field[]=title&proj_field[]=author).
+        in: query
+        schema:
+          type: string
+
+      responses:
+        200:
+          description: Release queue information and projects
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  queue:
+                    type: object
+                    $ref: '#/components/schemas/release_queue_detail'
+                  stats:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        days_ago:
+                          type: integer
+                        projects_released:
+                          type: integer
+                        pages_released:
+                          type: integer
+                  projects:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        project_id:
```

Comment:
This should be `projectid` (here and in the code) to match what the v1/projects endpoint returns.

Reply:
Fixed